tests/scripts/validate_cluster.sh

```diff
@@ -89,17 +89,20 @@ function test_demo_pool {
 
 function test_csi {
   timeout 360 bash -x <<-'EOF'
+    echo $IS_POD_NETWORK
     echo $IS_MULTUS
-    if [ -z $IS_MULTUS ]; then
+    if [ -z $IS_POD_NETWORK ]; then
       until [[ "$(kubectl -n rook-ceph get pods --field-selector=status.phase=Running|grep -c ^csi-)" -eq 4 ]]; do
         echo "waiting for csi pods to be ready"
         sleep 5
       done
     else
       until [[ "$(kubectl -n rook-ceph get pods --field-selector=status.phase=Running|grep -c ^csi-)" -eq 6 ]]; do
-        echo "waiting for csi pods to be ready with multus"
+        echo "waiting for csi pods to be ready with multus or pod networking"
         sleep 5
       done
+    fi
+    if [ -n $IS_MULTUS ]; then
```

Comment:
@leseb looks like with this change CI is failing.

Reply:
yes not sure why this fails, shall i revert back it back as the functionality remains same?

Reply:
pushing it again to check `! -z` behaviour

Reply:
Are we sure it's that change that makes the CI fail?

Reply:
Looks like it's failing again so `-n` was correct.

Reply:
The failure is something else. restarted to check it.